src/style.css

```diff
@@ -1 +1,127 @@
 /* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 600;
+}
+
+:root {
+  --main-color: #3d4e61;
+}
+
+html {
+  font-family: Avenir, sans-serif;
```

Comment:
Consider adding fallback font families to improve robustness in case the primary font fails to load. For example, you could use 'Avenir, Arial, sans-serif'.